components/select/src/vwc-select.ts

```diff
@@ -48,6 +48,9 @@ export class VWCSelect extends MWCSelect {
 	@property({ type: String, reflect: true })
 		name: string | undefined;
 
+	@property({ type: Boolean, reflect: true, attribute: 'ghost' })
```

Comment:
This needs to be tested.
Also, need to test the logic that's bound to it:
If ghost is true, dense must be true (at least that's what I see in the code).

Reply:
should we enforce programmatically? it's that mutating user's definition discussion again...

Reply:
We need because of the design. As I wrote in the summery of this PR its a quick & dirty fix. This component is different from our conventions in a lot of ways...